websites/A/AnimeLib/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "437154602626973697",
+		"name": "Kitosight"
+	},
+	"service": "AnimeLib",
+	"altnames": [
+		"AniLib"
+	],
+	"description": {
+		"en": "Watch anime online in Russian",
+		"ru": "Смотреть аниме онлайн на русском"
+	},
+	"url": "anilib.me",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/8vAN1DA.png",
+	"thumbnail": "https://i.imgur.com/W1iU6v0.png",
+	"color": "#A020F0",
+	"category": "anime",
+	"tags": [
+		"anime"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*",
```

Comment:
Are you able to make this regex more narrow?

Reply:
> Are you able to make this regex more narrow?

Sorry, but no
If you can say me how, I'll appreciate it.
I just have no any idea how to make it more narrow.

Reply:
All I could find is `kodik.info` tho

Reply:
> All I could find is `kodik.info` tho

That's what I was thinking about on my own, `"iFrameRegExp": "kodik\.info"`.

But, is that really so necessary?

Reply:
"kodik[.]info" and yes it is.